chain/bitcoind_zmq_events.go

```diff
@@ -399,6 +399,11 @@ func (b *bitcoindZMQEvents) mempoolPoller() {
 	for {
 		select {
 		case <-ticker.C:
+			log.Debugf("Reconcliing mempool spends with node " +
```

Comment:
Could make a sort of `logExecutionTime` func closure for this and the other instances.